app/javascript/controllers/select_autocomplete_controller.js

```diff
@@ -0,0 +1,15 @@
+import { Controller } from "@hotwired/stimulus"
+import accessibleAutocomplete from 'accessible-autocomplete'
+
+// Connects to data-controller="select-autocomplete"
+export default class extends Controller {
+  static targets = [ "input" ]
+
+  connect() {
+    accessibleAutocomplete.enhanceSelectElement({
+      selectElement: document.querySelector(
+        `#${this.inputTarget.id}`
+        )
```

Comment:
The indentation of this line is slightly off. Do we have a linter enabled for JS? 🤔

Reply:
We don't. We'd need to add in ESLint.

Also, out of curiosity, is there anything stopping us from doing...

```js
    accessibleAutocomplete.enhanceSelectElement({
      selectElement: this.inputTarget
    })
```